java/docker/java-builder/Dockerfile

```diff
@@ -96,9 +94,10 @@ RUN umask g+w \
  && ./gradlew :java:component:keyple-plugin:keyple-plugin-remotese:assemble --info \
  && ./gradlew -b ./android/build.gradle :keyple-plugin:keyple-plugin-android-nfc:uploadArchives \
  && ./gradlew -b ./java/example/calypso/android/nfc/build.gradle assembleDebug \
```

Comment:
jenkins use global gradle whereas docker uses gradlew, isn't it a problem?

Reply:
I see the Docker image in a bigger picture and I'd like the wrapper to be in gradle's cache.
But with Jenkins it's simpler not to use it.